src/app/core/callendar/transformers/default-data-transformer.ts

```diff
@@ -15,6 +15,8 @@ export class DefaultDataTransformer implements IcalendarTransformer<SessionViewM
         const sessionType = (session.sessionType) ? session.sessionType.description : 'No Session type';
 
         return {
+            person: session.person,
```

Comment:
Correct me if I'm wrong, but are we updating this fields after a resource has been changed?
for example Session A, is in Room1, we move it to Room2, now besides having proper resourceId in this transformed fullcalendar element are those fields also updated?

Reply:
yep, all events are rerender and has proper values :)